libcontainer/setns_init_linux.go

```diff
@@ -110,6 +110,11 @@ func (l *linuxSetnsInit) Init() error {
 	if err := apparmor.ApplyProfile(l.config.AppArmorProfile); err != nil {
 		return err
 	}
+	if l.config.Config.MemoryPolicy != nil {
+		if err := setupMemoryPolicy(l.config.Config); err != nil {
+			return err
+		}
+	}
```

Comment:
This should be done earlier -- at least before seccomp (so that users don't need to allow `set_mempolicy(2)` in their containers) and ideally when we are setting up other resources like cgroup limits.

Reply:
Great finding, thanks! Moved.